docs/configuring-metabase/config-file.md

```diff
@@ -132,6 +134,44 @@ config:
 
 See [Uploads](../databases/uploads.md).
 
+## API keys
+
+You can use the config file to create API keys, which is useful for automated deployments and keeping API keys stable across environments.
+
+You can add API keys like so:
+
+```yaml
+{% raw %}
+version: 1
+config:
+  users:
+    - first_name: Cam
+      last_name: Era
+      password: 2cans3cans4cans
+      email: cam@example.com
+  api-keys:
+    - name: "Admin API key"
+      group: admin
+      description: "API key with admin permissions."
+      creator: cam@example.com
+      key: mb_firsttestapikey
+    - name: "All Users API key"
+      group: all-users
+      description: "API key with permissions of the All Users group."
+      creator: cam@example.com
+      key: mb_secondtestapikey
+{% endraw %}
+```
+Some things to know about creating API keys with the config file:
+
+- API keys that you create (the value of the `key`) must have the format `mb_` followed by a [Base64](https://en.wikipedia.org/wiki/Base64) string (if you're wearing formal attire, you'd say a _tetrasexagesimal_ string). So, `mb_` followed by letters and numbers. Concretely, the API key you create must satisfy the following regular expression: `mb_[A-Za-z0-9+/=]+`.
+- Only admins can create API keys. This means either a) the Metabase must already have at least one admin account, or b) you need to add an admin account in the `users` section of the config file.
+- The keys themselves can be assigned to non-admin groups. Meaning: the permissions for the key are defined by the `group`, not the `creator`.
```

Comment:
It is relevant to mention that api-keys in the config file can only be assigned to the admin or all-users group. That's because these are the only groups that we know for sure exist on startup